x/protocolpool/keeper/keeper.go

```diff
@@ -19,7 +22,13 @@ type Keeper struct {
 	authKeeper   types.AccountKeeper
 	bankKeeper   types.BankKeeper
 
+	cdc codec.BinaryCodec
+
 	authority string
+
+	// State
+	Schema         collections.Schema
+	BudgetProposal collections.Map[sdk.AccAddress, types.MsgBudgetProposal]
 }
```

Comment:
The keeper misses an authority.

Reply:
I'm not sure if I understand this correctly, but authority is already defined in keeper at L27

Reply:
Lol, right, my bad.